asyncio_redis_rate_limit/__init__.py

```diff
@@ -95,7 +99,17 @@ async def _acquire(self) -> None:
         pipeline = self._backend.pipeline()
 
         async with self._lock:
-            current_rate = await self._run_pipeline(cache_key, pipeline)
+            redis_version = await self.redis_version()
+
+            if int(redis_version.split('.')[0]) < 7:
+                current_rate, *_ = await pipeline.incr(cache_key).execute()
```

Comment:
It feels like you execute a pipeline twice. This might not be safe due to race conditions.